docs/Configuration.md

```diff
@@ -361,3 +361,39 @@ dotnet stryker -gs "development"
 ```
 
 Default: `master`
+
+
+## Dashboard Compare
+Enabling the dashboard compare feature saves reports and re-uses the result when a mutant or it's tests are unchanged.
+
+```
+dotnet stryker --dashboard-compare
+dotnet stryker -compare
+```
+
+Default `"off"`
+
+This feature automatically enables the --diff feature.
+
+## Fallback version
+When enabling the --dashboard-compare feature you can provide a fallback version. This version will be used to pull a baseline when we cannot find a baseline for your current branch. When we are still unable to provide a baseline we will start a complete testrun to create a complete baseline.
+
+```
+dotnet stryker --dashboard-compare --dashboard-fallback-version master
+dotnet stryker -compare -fallback-version master
+```
+Default: `null`
+
+## Configuring Dashboard Compare on pull requests
+When configuring the --dashboard-compare feature on pull requests please provide the following configurations.
+
+1.  Enable --dashboard-compare.
+2. Set --dashboard-version to the name of the source branch for your pull request.
+3. Set --dashboard-fallback-version to the name of the target branch for your pull request.
```

Comment:
We should allow for only a fallback version or only a git source to be set. If only one is set, it should be used for both. If we recommend to use the same value for both, we should handle that case as painless as possible.

Reply:
Right now GitSource is not allowed to be null, this will result in an exception. I have added a check in StrykerOptions that checks if FallbackVersion is null. If this is the case it is set to GitSource.